app/models/cohort.rb

```diff
@@ -1,41 +1,7 @@
 class Cohort < ActiveRecord::Base
   has_many :groups
 
-  def homepage_visits
-    Ahoy::Event.where(name: '$view')
-               .where("properties ->> 'url'= 'https://www.loomio.org/'")
-               .where(time: start_on..(end_on + 1.day))
-  end
-
   def organisations
     groups.where('parent_id IS NULL')
   end
-
-  def activated_organisations
-    # organisation_visits_count
```

Comment:
@HSalmon would you mind taking this one? #3045 

I think you just have to put this method back in.